econplayground/templates/main/cohort_form.html

```diff
@@ -26,7 +26,7 @@
 {% endblock %}
 
 {% block pagetitle %}
-    {% if object %}Edit {% else %}New {% endif %} Cohort
+    {% if object %}Edit {% else %}New {% endif %} Course
```

Comment:
You're changing Cohort to Course here, but leaving it as Cohort earlier on in this file. Let's stick with either Cohort, or Course on the front-end, but not both.

Reply:
Forgot to change that one Cohort, thanks